shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -45,7 +45,13 @@ static bool ClipRectContainsPlatformViewBoundingRect(const SkRect& clip_rect,
                                                      const SkRect& platformview_boundingrect,
                                                      const SkMatrix& transform_matrix) {
   SkRect transformed_rect = transform_matrix.mapRect(clip_rect);
-  return transformed_rect.contains(platformview_boundingrect);
+
+  // Tolerate floating point errors when comparing flow clipping view bounds
+  // and quartz platform view bounds. Round to integers.
```

Comment:
Physical pixels since it's running right before this logic:
https://github.com/flutter/engine/blob/45137ea6ba29f7bb9f7d653ee3efeee432128df3/shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm#L562-L567

Reply:
if its physical pixels and the error is at the scale of 0.00001, then even rounding to the nearest 0.01 should resolve the problem without having any meaningful effect on the rendering while still being way smaller than the space between two sample points (at 4x MSAA there are 4 per physical pixel).

Reply:
I think it should be fine to copy the logic here and round the outer rect out:
https://github.com/flutter/engine/blob/919af2c3155dc33873622ebe42794e8fdb3a8009/shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm#L730-L733
```c++
    /** Sets SkRect by discarding the fractional portion of fLeft and fTop; and rounding
        up fRight and fBottom, using
        (sk_float_floor(fLeft), sk_float_floor(fTop),
         sk_float_ceil(fRight), sk_float_ceil(fBottom)).

        @param dst  storage for SkRect
    */
    void roundOut(SkRect* dst) const {
        dst->setLTRB(sk_float_floor(fLeft), sk_float_floor(fTop),
                     sk_float_ceil(fRight), sk_float_ceil(fBottom));
    }
```